typings.json

```diff
@@ -5,6 +5,7 @@
     "tape": "github:DefinitelyTyped/DefinitelyTyped/tape/tape.d.ts#221df9ad820d0b2aa010d328c4bab06c8d69ac1f"
   },
   "dependencies": {
+    "chai": "registry:npm/chai#3.5.0+20160328084239",
```

Comment:
I added Chai to the typings because my project got all mad that it wasn't able to find chai. Is this supposed to be pulled in as a dependency or something?